core/src/main/java/org/elasticsearch/rest/action/support/RestActions.java

```diff
@@ -64,18 +73,24 @@ public static long parseVersion(RestRequest request, long defaultVersion) {
     }
 
     static final class Fields {
+        static final String _NODES = "_nodes";
```

Comment:
One day Java will add a true `const` and we can actually use these types of constructs like we're trying to do here: ensure consistency without the bloat of adding yet another class / static reference.

But not this day.

I went ahead and removed them, but I do think that these things actually add value when the field is used more than once in the class, but it's not worth the cost (we really do have enough classes without adding noop ones like these).